qlabfb.js

```diff
@@ -1,59 +1,72 @@
 /* eslint-disable no-useless-escape */
 var instance_skel = require('../../instance_skel');
+var rgb = require('../../image').rgb;
+var presets = require('./presets');
+var actions = require('./actions');
+var variables = require('./variables');
+var feedbacks = require('./feedbacks');
+var Cue = require('./cues');
+var OSC = require('osc');
+//var util = require('util');
 var debug;
-// eslint-disable-next-line no-unused-vars
+
 var log;
-var OSC = require('osc');
 
-// eslint-disable-next-line no-unused-vars
 function instance(system, id, config) {
 	var self = this;
 	var po = 0;
+
 	self.connecting = false;
 	self.needPasscode = false;
 	self.useTCP = config.useTCP;
 	self.qLab3 = false;
 	self.hasError = false;
-	self.diabled = true;
-
+	self.disabled = true;
 	self.ws = '';
+	self.pollCount = 0;
 
 	self.resetVars();
 
-	self.colorRGB = {
-		none: 	self.rgb(32, 32, 32),
-		red: 	self.rgb(160, 0, 0),
-		orange: self.rgb(160, 100, 0),
-		green: 	self.rgb(0, 160, 0),
-		blue: 	self.rgb(0, 0, 160),
-		purple: self.rgb(160, 0, 160)
-	};
-
 	// each instance needs a separate local port
 	id.split('').forEach(function (c) {
 		po += c.charCodeAt(0);
 	});
-
 	self.port_offset = po;
 
 	// super-constructor
 	instance_skel.apply(this, arguments);
 
 	self.actions(); // export actions
+
+	if (process.env.DEVELOPER) {
+		self.config._configIdx = -1;
+	}
+
 	self.addUpgradeScript(function (config, actions) {
```

Comment:
@istnv since you're adding the release actions to this script, you need to include that parameter in the function: `self.addUpgradeScript(function (config, actions, releaseActions, feedbacks) {`

Reply:
Done. Thanks. 
I thought there might be something I missed, but I've been staring at so much code lately :)